web/modules/custom/rdf_entity/rdf_taxonomy/src/TermRdfStorage.php

```diff
@@ -208,7 +208,7 @@ public function loadTree($vid, $parent = 0, $max_depth = NULL, $load_entities =
       if (empty($this->treeChildren[$vid])) {
         /** @var \Drupal\taxonomy\Entity\Vocabulary $voc */
         $voc = entity_load('taxonomy_vocabulary', $vid);
-        $settings = $voc->getThirdPartySetting('rdf_entity', 'mapping_vid');
+        $settings = $voc->getThirdPartySetting('rdf_entity', 'mapping')['vid'];
```

Comment:
Again, how come the vocabulary has a direct call to the getThirdPartySettings and the rdf entity has to call 'rdf_entity_get_third_party_property' instead?
Can we use the service in both cases here?
We have config/content entity properties and field properties only.